tracee-ebpf/tracee/tracee.go

```diff
@@ -1037,10 +1075,23 @@ func (t *Tracee) updateFileSHA() {
 	}
 }
 
-func (t *Tracee) invokeInitNamespacesEvent() {
```

Comment:
@yanivagman 
Because the hooked syscalls detection also start at the user space (by trigger the IOCTL technique ) and the hooked file operations event also will be trigger by that I thought to change `t.invokeInitNamespacesEvent` to `t.invokeManualTriggeredEvents` and there include the whole events that we will start or create in the user space. 
 what you think about it ?

Reply:
We should keep those decoupled:
1. Trigger ioctl events when required (doesn't have to be during init)
2. Trigger init related "system state probing" - this includes init namespaces event (which is not exactly an event) and other events in the future, such as process tree init state, container init state, etc.

Reply:
Ok, ​i will roll it back and add function for handling IOCTL driven event.
​I think just for now it will be fine to name it as `invokeSyscallDetection` and later on when the fops detection will be in ill rename it for handling the all IOCTL driven even